.github/workflows/webviz-subsurface-components.yml

```diff
@@ -41,10 +41,10 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Use Node.js 15.12
-        uses: actions/setup-node@v1
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
         with:
-          node-version: "15.12"
+          node-version: 18
```

Comment:
Should be < 18, until https://github.com/storybookjs/storybook/issues/18019 (as @shadab-skhan mentioned) is fixed

Reply:
:+1: Changed to `node == 16` in CI.